homeassistant/components/apsystems/__init__.py

```diff
@@ -12,15 +14,26 @@
 
 PLATFORMS: list[Platform] = [Platform.SENSOR]
 
-ApsystemsConfigEntry = ConfigEntry[ApSystemsDataCoordinator]
+ApSystemsConfigEntry = ConfigEntry["ApSystemsData"]
```

Comment:
```suggestion
type ApSystemsConfigEntry = ConfigEntry[ApSystemsData]
```
From #117630 / #117632

Reply:
Nice, but mypy isn't up to date enough to know that, so I cannot run any mypy tests anymore.